src/library/searchquery.h

```diff
@@ -16,6 +16,8 @@
 #include "util/memory.h"
 #include "library/crate/cratestorage.h"
 
+const QString kExpliciteEmpty = "\"\""; // "" searches for an empty string
```

Comment:
The name of the constant is not a valid English term and needs to be explained. The comment is misleading, because it applies to all kind of fields and not only strings.

- Maybe kMissingFieldSearchTerm or something else?
- The SQL uses "IS NULL" which does not match fields with *empty strings*!? What is the desired result for string fields? I suggest to cover both NULL and empty strings with this filter.